tests/bootstrap.php

```diff
@@ -25,6 +25,7 @@
 define('CAKE', CORE_PATH . APP_DIR . DS);
 
 require dirname(__DIR__) . '/vendor/autoload.php';
+require CONFIG . 'bootstrap.php';
```

Comment:
should this be after the core one maybe? or is the order ok like this?

Reply:
> should this be after the core one maybe? or is the order ok like this?

Let's move it to be safe